doc/guides/modules.rst

```diff
@@ -10,14 +10,380 @@ system those are called *modules*. These modules must be integrated with the
+the incoming changes are always **reviewable**, and the  *downstream* module
+repository history is incremental and always preserved (that is, it is never
+re-written). This policy also allows to run Zephyr CI, git lint, identity, and
+license checks directly on the set of changes that are to be brought into the
+module repository.
+
+.. note::
+
+     Force-pushing to a module's main branch is not allowed.
+
+Allowed practices
+---------------------
+
+The following practices conform to the above requirements and should be
+followed in all modules repositories. It is up to the module code owner
+to select the preferred synchronization practice, however, it is required
+that the selected practice is consistently followed in the respective
+module repository.
+
+**Updating modules with a diff from upstream:**
+Upstream changes brought as a single *snapshot* commit (manual diff) in a
+pull request against the module's main branch. This approach is simple and
+should be applicable to all modules with the downside of supressing the
+upstream history in the module repository.
+
+  .. note::
+
+     The above practice is the only allowed practice in modules where
+     the external project is not hosted in an upstream Git repository.
+
+The commit message is expected to identify the upstream repository, the
+version to which the module is updated (upstream version, tag, commit,
+etc.), and the reason for the doing the update.
+
+**Updating modules by merging the upstream main branch:**
+Upstream changes brought in by performing a Git merge of the upstream main
+branch submitting the result in pull request against the module main branch.
+This approach is applicable to modules with an upstream project Git repository.
+The main advantages of this approach is that the upstream repository history
+(that is, the original commit SHAs) is preserved in the module repository. The
+downside of this approach is that an additional merge commit is generated in
+the downstream main branch.
+
+
+Contributing to Zephyr modules
+******************************
+
+.. _modules_contributing:
+
+
+Individual Roles & Responsibilities
+===================================
+
+To facilitate management of Zephyr module repositories, the following
+individual roles are defined.
+
+**Administrator:** Each Zephyr module shall have an administrator
+who is responsible for managing access to the module repository,
+for example, for adding individuals as Collaborators in the repository
+at the request of the module owner. Module administrators are
+members of the Administrators team, that is a group of project
+members with admin rights to Module Github repositories.
+
+**Module owner:** Each module shall have a module code owner. Module
+owners will have the overall responsibility of the contents of a
+Zephyr module repository. In particular, a module owner will:
+
+* coordinate code reviewing in the module repository
```

Comment:
I guess the extra context here is that the Owner will moderate the code reviewing (besides being Assignee and requesting code reviews).
Yes, it might overlap a bit, but no real harm keeping it for now, as this is what we agreed upon :)